src/drivers/libinput/lv_xkb.c

```diff
@@ -120,6 +120,10 @@ uint32_t lv_xkb_process_key(lv_xkb_t * dsc, uint32_t scancode, bool down)
         case XKB_KEY_ISO_Left_Tab: /* Sent on SHIFT + TAB */
             result = LV_KEY_PREV;
             break;
+        case XKB_KEY_Home:
+        case XKB_KEY_KP_Home:
+            result = LV_KEY_HOME;
+            break
```

Comment:
need `;` at the end

Reply:
Oops, I guess it was better to start the weekend. ;)